### DISCUSSION TITLE
some kind of error

### DISCUSSION THREAD

**Discussion by n1kkilou** [GriefPrevention/GriefPrevention#1566] [General]
can anyone help me with this error

GriefPrevention.jar//me.ryanhamshire.GriefPrevention.PlayerEventHandler.onPlayerInteract(PlayerEventHandler.java:1692)

**Comment by bigpresh** [comment]
Please [raise an issue](https://github.com/TechFortress/GriefPrevention/issues/new/choose), filling out the issue template with the details it asks for, and we'll see what we can do to help.
